meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor.bb

```diff
@@ -72,7 +72,8 @@ api_fetch_supervisor_image() {
 }
 
 do_install () {
-	SUPERVISOR_IMAGE=$(api_fetch_supervisor_image "${SUPERVISOR_VERSION}")
+	# SUPERVISOR_IMAGE=$(api_fetch_supervisor_image "${SUPERVISOR_VERSION}")
+	SUPERVISOR_IMAGE="registry2.balena-cloud.com/v2/f356794f24dd1c254f9348a36d4fc9a5"
```

Comment:
The preload actually happens here:
https://github.com/balena-os/meta-balena/blob/master/meta-balena-common/recipes-containers/docker-disk/files/entry.sh#L64-L75

Reply:
To be clear, you need the change in both places.
1. balena-supervisor.bb is more of a sanity check
2. docker-disk does the actual preloading

Reply:
oh, thanks for the pointer :)